src/murmur/Server.cpp

```diff
@@ -460,6 +461,11 @@ void Server::readParams() {
 	iDefaultChan = getConf("defaultchannel", iDefaultChan).toInt();
 	bRememberChan = getConf("rememberchannel", bRememberChan).toBool();
 	qsWelcomeText = getConf("welcometext", qsWelcomeText).toString();
+	qsWelcomeTextFile = getConf("welcometextfile", qsWelcomeTextFile).toString();
+
+	if (!qsWelcomeTextFile.isEmpty()) {
+		loadWelcomeTextFromFile();
```

Comment:
I think I'd prefer if this function would take the file to read from as an argument instead of relying on certain variables having certain values :point_up:

Reply:
I have removed the function, since it is only called once.